automated-e2e-test/env_vars/SODA_Globals.postman_globals.json

```diff
@@ -0,0 +1,59 @@
+{
+	"id": "b2f43c3a-bb53-49d8-8402-d8d597f2b534",
+	"values": [
+		{
+			"key": "authToken",
+			"value": "gAAAAABfVh5ocViaWyubzkDJaVtTWZ0iaL3V0C-h6VQQn_LkrtdQ3f25mbiyJrNDDzlggvTGg0ORcmm5t2iWyKfr9dGPsXeQqktepNBHTK6EALaDGY6xuaix41LiGhHGUdopfwwKT...",
+			"enabled": true
+		},
+		{
+			"key": "HOST_IP",
+			"value": "192.168.56.128",
```

Comment:
replace actual ip with placeholder

Reply:
I will change it to the loopback IP address, `127.0.0.1` since SODA default installation also uses this IP address.